examples/sessions/templates/index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>{% block title %}{% endblock %}</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  {% block head %}
+    <link rel="stylesheet" href="https://unpkg.com/tachyons@4.12.0/css/tachyons.min.css"/>
+  {% endblock %}
+</head>
+<body class="sans-serif">
+  {% block body %}
+    <div id="main" class="center mw7 center">
+        <h1>Boltzmann sessions example</h1>
+        {% if user %}
```

Comment:
We might add a template comment (`{# #}`, IIRC?) here noting that we're adding `user` to context in the `templateContext` middleware attachment in `middleware.js#L25`.